### PR TITLE
Heartbeat API

### PR DESCRIPTION
**Pull request by anthonypetersen**
This PR addresses the following Issue:
* N/A
-----
## Background Details
* Adding foundation for Connect Heartbeat API
-----
## Technical Changes
* Added new API end-point
* Updated YAML files
* Consolidated `require` statement for notifications in `index.js`
* Fixed spelling of function

### CODE REVIEW

File: utils/heartbeat.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+const { logIPAdddress, setHeaders } = require('./shared');
+
+const heartbeat = async (req, res) => {
+    logIPAdddress(req);
```
Comment (JoeArmani) [inline utils/heartbeat.js:4]: small typo I think

File: config/stage/cloudbuild1.yaml [hunk 2]
```diff
@@ -7,6 +7,10 @@ steps:
   args: ['functions', 'deploy', 'biospecimen', '--trigger-http', '--runtime=${_RUNTIME}', '--source=${_SOURCE}', '--env-vars-file=config/stage/.env.yaml']
 - name: 'gcr.io/cloud-builders/gcloud'
   args: ['functions', 'add-iam-policy-binding', 'biospecimen', '--member=allUsers', '--role=${_ROLE}']
+- name: 'gcr.io/cloud-builders/gcloud'
+  args: ['functions', 'deploy', 'heartbeat', '--trigger-http', '--runtime=${_RUNTIME}', '--source=${_SOURCE}', '--env-vars-file=config/prod/.env.yaml']
```
Comment (JoeArmani) [inline config/stage/cloudbuild1.yaml:11]: do we want the stage path here? `config/stage/.env.yaml`

File: config/dev/cloudbuild1.yaml [hunk 3]
```diff
@@ -7,6 +7,10 @@ steps:
   args: ['functions', 'deploy', 'biospecimen', '--trigger-http', '--runtime=${_RUNTIME}', '--source=${_SOURCE}', '--env-vars-file=config/dev/.env.yaml']
 - name: 'gcr.io/cloud-builders/gcloud'
   args: ['functions', 'add-iam-policy-binding', 'biospecimen', '--member=allUsers', '--role=${_ROLE}']
+- name: 'gcr.io/cloud-builders/gcloud'
+  args: ['functions', 'deploy', 'heartbeat', '--trigger-http', '--runtime=${_RUNTIME}', '--source=${_SOURCE}', '--env-vars-file=config/prod/.env.yaml']
```
Comment (JoeArmani) [inline config/dev/cloudbuild1.yaml:11]: do we want the dev path here? `config/dev/.env.yaml`

**Review by JoeArmani — APPROVED** [review]
looks good

left minor comments

File: utils/heartbeat.js (hunk 1 above)
Comment (anthonypetersen) [reply utils/heartbeat.js:4]: unfortunately that's the "right" spelling, but I'll correct it

**Review by we-ai — APPROVED** [review]

**Pull request merged**